web/pandas/community/ecosystem.md

```diff
@@ -345,11 +345,12 @@ which pandas excels.
 
 ## IO
 
-### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas)*: A semantic, compact and reversible JSON-pandas converter*
+### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas): *A semantic, compact and reversible JSON-pandas converter*
 
-NTV-pandas provides a JSON converter with more data types than the ones supported by pandas directly.
+`NTV-pandas` provides a JSON converter with more data types than the ones supported by pandas directly.
 
 It supports the following data types:
+
 - pandas data types
 - data types defined in the [NTV format](https://loco-philippe.github.io/ES/JSON%20semantic%20format%20(JSON-NTV).htm)
 - data types defined in [Table Schema specification](http://dataprotocols.org/json-table-schema/#field-types-and-formats)
```

Comment:
Couple of lines before, can you change `*example*` (in itallics) by simply `Example:`, and also leave a blank line between that and the code block please?

And inside the block, can you leave two spaces between the code and the comment? In one line it's already like this, but can you add the missing spaces to the other two, so it's consistent and follows PEP-8. And if you want, you can surround `DataFrame`, `True` and `table=True` by blackticks, even if it won't render, as those are code inside the text.